docs/topics/ksp/ksp-overview.md

```diff
@@ -1,53 +1,43 @@
 [//]: # (title: Kotlin Symbol Processing API)
 
-Kotlin Symbol Processing (_KSP_) is an API that you can use to develop
-lightweight compiler plugins. KSP provides a simplified compiler plugin
-API that leverages the power of Kotlin while keeping the learning curve at
-a minimum. Compared to KAPT, annotation processors that use KSP can run up to 2x faster.
+Kotlin Symbol Processing (_KSP_) is an API that you can use to develop lightweight compiler plugins. 
+KSP provides a simplified compiler plugin API that leverages the power of Kotlin while keeping the learning curve at
+a minimum. Compared to [kapt](kapt.md), annotation processors that use KSP can run up to 2 times faster.
 
-To learn more about how KSP compares to KAPT, check out [why KSP](ksp-why-ksp.md). To get started writing a KSP processor, take a look at the [KSP quickstart](ksp-quickstart.md).
+To learn more about how KSP compares to kapt, check out [why KSP](ksp-why-ksp.md). 
```

Comment:
There is a capitalized KAPT in the ksp-why-ksp.md

Reply:
Wops, it looks like I've missed this file. I'm lucky that everything is fine there.

Reply:
Thanks for noticing. Reformatted.